scanpy_scripts/cli.py

```diff
@@ -74,7 +75,7 @@ def cli(debug=False, verbosity=3):
 cli.add_command(REGRESS_CMD)
 cli.add_command(PCA_CMD)
 cli.add_command(NEIGHBOR_CMD)
-
+cli.add_command(COMBAT_CMD)
```

Comment:
If we are to include multiple batch correction methods in scanpy-scripts in the future, shall we have a batch correction "umbrella" command (just like embed, plot, etc) and place combat under that?